states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Variables for Menu options"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays options for Menu state change"""
+
     option: Optional[MenuOptions] = None
 
     def __init__(self):
+        """Logic for what occurs if exit option is not choesen"""
```

Comment:
You're right, but we might handle the exit option differently in the future. Talk about how this initializes the menu state instead and lists options, then checks options for what a player selects.

Reply:
Also minor: `chosen` not `choesen`

Reply:
Changed to `Initializes the menu state and lists available options checking against user input if exit is not chosen`